specification/e-referrals-service-api.yaml

```diff
@@ -217,6 +217,8 @@ info:
   license:
     name: MIT
 servers:
+  - url: 'https://internal-dev-sandbox.api.service.nhs.uk/referrals-pr-145/FHIR'
```

Comment:
Question? Do we need to remove the PR URL after testing is done.?

Reply:
Yes